src/lint/lint.ts

```diff
@@ -87,3 +104,64 @@ export async function lint(options: Options, config: ProjectConfig) {
     return new CommandResult(1);
   }
 }
+
+async function fix(
+    warnings: FixableWarning[],
+    options: Options,
+    config: ProjectConfig,
+    analyzer: Analyzer,
+    analysis: Analysis) {
+  if (options.input) {
```

Comment:
should this be handled outside of the function? If I gave a set of warnings to a `fix()` function to fix, I would expect them all to be fixed/reported. At the very least, document this to make expected behavior clear.

Reply:
+1, and better to be more consistent between reporting and fixing